### ISSUE TITLE
roadJuntion.fmw?

### ISSUE THREAD

**Issue by sab56** [OrdnanceSurvey/OSMM-Highways-Network-Support#12]
Should that be roadJunction.fmw?

**Comment by ibennett** [comment]
Yes a RoadJunction feature is a collection of RoadNode features that share the same road junction name or number. The collection of RoadNodes will include all RoadNodes that constitute part of a “slip road” or other link between two or more crossing road features (Figure 12). A RoadNode could be referenced by multiple RoadJuntion features. Motorway Junctions will be populated completely, other junctions are being supplied as data is improved for future releases.
